Add module READMEs and update main README (#11)
WayScience/JUMP-single-cell: 2.evaluate_data/README.md

# Apply phenotypic profiling model to JUMP data

In this module, we generate single-cell probabilities for each of the 15 phenotypic classes by applying the [phenotypic profiling model](https://github.com/WayScience/phenotypic_profiling_model).

> There are two model types, final and shuffled baseline, where single-cell features are randomized independently. 

Reviewer: ```suggestion
There are two model types, final and shuffled baseline.
The shuffled baseline model trains using randomly shuffled single-cell features. 
```

To increase clarity and move to active voice.

Author: 👍 changed!

> We output one probability `parquet` file that contains all of the single-cell data per plate and well that contains all relevant metadata.

Reviewer: ```suggestion
We output one file per plate that contains phenotypic probabilities and relevant metadata for all of the single-cells.
The files we output are in `parquet` format.
```

Reviewer: For clarity

Author: @MattsonCam Can you clarify if the "predict_sc_probabilities.ipynb" notebook outputs the model predictions as one file with all plates or a file per plate. Based on what I can see, it looks like the first option:

```python
        # Store the prediction dataframes
        model_preds.append(predsdf)

# Concatenate the model predictions
model_preds = pd.concat(model_preds)

# Save predictions
model_preds.to_parquet(f"{output_proba_path}/model_probabilities.parquet", index=True)
```
Once this is clarified I can for sure make this change.

Reviewer 2: Good question @jenna-tomkinson, I only create one parquet file for all of the predicted probabilities for now

Author: Thanks @MattsonCam for clarifying! Then I will keep the wording I had in place.

## Run the prediction notebook

To generate the probabilities for each single cell, run the [evaluate_data.sh](./evaluate_data.sh) file which will convert the notebook into a python file and run it from terminal.

```bash
# Make sure you are in the 2.evaluate_data directory
cd 2.evaluate_data
# Run the notebook as a python script
source evaluate_data.sh
```

> 

Reviewer: ```suggestion
```
Delete extra line

Author: Thanks for catching this! Changed.
